### PR TITLE
Feature/us 03

### PR DESCRIPTION
**Pull request by fel-sad**
Pull Request for add an event user story. Please not that the "picture" requires at the moment an URL from unsplashed - an inproper URL will result in an error from next Image.
(Data Validation / image upload is planned for a later user story)

### CODE REVIEW

File: components/EventForm.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  return (
+    <>
+      <StyledForm onSubmit={handleSubmit}>
+        <h2>Add an event</h2>
+        <label htmlFor="title">Title*</label>
+        <input
+          id="title"
+          name="title"
+          aria-label="Event Title"
+          placeholder="Was is your event called?"
```
Comment (alhen002) [inline components/EventForm.js:36]: What*

File: components/EventForm.js
Comment (alhen002) [inline components/EventForm.js:1]: i would suggest adding the aria-labeldby for all your inputs and labels:
1. 
`<label id="expLabel" for="expire">Expiration date:</label>
<span>
	<input type="text" name="expire" id="expire" aria-labelledby="expLabel expDesc">
	<span id="expDesc">MM/YYYY</span>
</span>
`
2. also thinks of semantic structure of your form with `<fieldset>` and `<legend>`for give your form more structure.

File: pages/api/events/index.js [hunk 2]
```diff
@@ -16,6 +16,16 @@ export default async function handler(request, response) {
       } catch (error) {
         return response.status(400).json({ message: error.message });
       }
+
+    case "POST":
+      try {
+        const newEvent = request.body;
+        await Event.create(newEvent);
+        response.status(201).json({ status: "Event created" });
```
Comment (alhen002) [inline pages/api/events/index.js:24]: `.json({ status / message: `Event with id ${newEvent._id} was created.` })`

**Review by alhen002 — COMMENTED** [review]
see my comments..

File: components/EventForm.js
Comment (fel-sad) [reply components/EventForm.js:1]: Implemented. :)

File: pages/api/events/index.js [hunk 3]
```diff
@@ -16,6 +16,16 @@ export default async function handler(request, response) {
       } catch (error) {
         return response.status(400).json({ message: error.message });
       }
+
+    case "POST":
+      try {
+        const newEvent = request.body;
+        await Event.create(newEvent);
```
[the remark this answers is not in the document]
Comment (alhen002) [reply pages/api/events/index.js:23]: This won't work. Please see my above comment.

[the remark this answers is not in the document]
Comment (fel-sad) [reply pages/api/events/index.js]: Thank you for catching this!  Updated!

**Review by alhen002 — APPROVED** [review]
lgtm:)

**Pull request merged**